app/helpers/application_helper.rb

```diff
@@ -166,7 +165,7 @@ def popover_for_user_name(user, team = nil, skip_user_status = false)
     end
 
     user_name = user.full_name
- 
+
```

Comment:
Maybe also consider removing `SmartAnnotations::TagToHtml.new(current_user, text).html`, just to be safe for the future. Only if it's possible/OK to do so.

Reply:
Looks like ApplicationHelper is included only in controllers, so current_user should be available there, I assume we can leave it as it is for now.